extensions/ql-vscode/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## [UNRELEASED]
 
+- Ensure databases are unlocked when removing them from the workspace. This will ensure that queries can be run on a database from the command line after being removed. Requires CodeQL CLI 2.4.1 or later. [#681](https://github.com/github/vscode-codeql/pull/681)
```

Comment:
That's not quite what we're getting at. When a database is in the workspace, you can't run queries from the command line. You must remove from vscode to run command line queries. How about this: 

```suggestion
- Ensure databases are unlocked when removing them from the workspace. This will ensure that after a database is removed from VS Code, queries can be run on it from the command line without restarting VS Code. Requires CodeQL CLI 2.4.1 or later. [#681](https://github.com/github/vscode-codeql/pull/681)
```